tools/deploy-to-master-stable-branch.sh

```diff
@@ -42,7 +42,6 @@ echo ""
 echo "Building Jetpack"
 npm install -g yarn@0.16.1
 yarn run distclean
-yarn cache clean
```

Comment:
Oh, messed up, you're right. But doesn't that have to be `yarn client-clean` then? You mean we need to drop everything that we have in `_inc/client`?